obspy/signal/src/pk_mbaer.c

```diff
@@ -265,10 +272,13 @@ int ppick (float *reltrc, int npts, int *pptime, char *pfm, float samplespersec,
       edat= edat*edat;         /* corresponds to SF(t) */
       omega= y2/yt;
 
-      if(sdev > 0)
+      if(sdev > 0) {
           edev= (edat-mean)/sdev;  /* corresponds to CF(t), mean corresponds
                                       to S(t)
                                    */
+          CF[CF_COUNTER]=edev ;    //MB
+          CF_COUNTER++ ;           //MB
+      }                            //MB add brackets {}.
```

Comment:
Please lowercase everything and use `/* */` style comments to retain a consistent code style in the file.

Reply:
Don't need the comment about brackets, nor the `MB` ones; `git blame` can provide that information.